go-controller/pkg/ovn/hybrid.go

```diff
@@ -126,9 +124,12 @@ func (oc *Controller) handleHybridOverlayPort(node *kapi.Node, annotator kube.An
 		}
 	}
 
+	if _, drIP := node.Annotations[hotypes.HybridOverlayDRIP]; !drIP {
```

Comment:
@JacobTanenbaum you could just grab the DRIP here, and then use it just below instead of getting it again? eg

```
drIP, ok := node.Annotations[hotypes.HybridOverlayDRIP]
if !ok {
...
}

if err := oc.setupHybridLRPolicySharedGw(subnets, node.Name, portMAC, net.ParseIP(drIP));
 != nil {
```

But also, do we need to validate the DRIP here too?

Reply:
I changed so I only need to grab the drIP once